pom.xml

```diff
@@ -68,45 +50,25 @@ under the License.
 	</modules>
 
 	<properties>
-		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-
 		<flink.version>1.16-SNAPSHOT</flink.version>
 		<flink.shaded.version>15.0</flink.shaded.version>
 
-		<!-- Overwrite default values from parent pom.
-			 Intellij is (sometimes?) using those values to choose target language level
-			 and thus is changing back to java 1.6 on each maven re-import -->
-		<target.java.version>1.8</target.java.version>
-		<maven.compiler.source>${target.java.version}</maven.compiler.source>
-		<maven.compiler.target>${target.java.version}</maven.compiler.target>
-
 		<junit4.version>4.13.2</junit4.version>
 		<junit5.version>5.8.1</junit5.version>
 		<assertj.version>3.21.0</assertj.version>
 		<archunit.version>0.22.0</archunit.version>
 		<testcontainers.version>1.17.2</testcontainers.version>
 		<mockito.version>2.21.0</mockito.version>
-		<spotless.version>2.13.0</spotless.version>
```

Comment:
This was upgraded, which also allows it to run on Java 17 (hence its no longer disabled in that profile)